products/nothing-phone.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Nothing Phone
+category: device
+tags: mobile-phone
+permalink: /nothing-phone
+versionCommand: "Settings -> About Phone"
+releaseColumn: false
+releaseDateColumn: true
+discontinuedColumn: true
+eoasColumn: Android Updates
+eolColumn: Security Updates
+customColumns:
+-   property: supportedAndroidVersions
+    position: after-release-column
+    label: Supported Android
+    description: Supported Android versions range
+    link: https://endoflife.date/android
```

Comment:
This link should ideally go to a table where this is documented for the entire range. Since we don't have such a link, lets drop it.
```suggestion
```

Reply:
I've took the custom column from [products/pixel.md#L19](https://github.com/endoflife-date/endoflife.date/blob/master/products/pixel.md?plain=1#L19).  
The idea was to link to the Android OS page for further details regarding the Android lifecycle.